cmd/broadcast/utils.go

```diff
@@ -0,0 +1,173 @@
+package main
+
+import (
+	"bufio"
+	"encoding/hex"
+	ida "github.com/harmony-one/libunison/internal/ida/coopcast"
+	"io"
+	"log"
+	"math/rand"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const pubKeySize int = 20
+
+// Entry is a single config of a node.
+type Entry struct {
```

Comment:
If that is the case, I guess a more apt name is `PeerInfo` or `PeerConfig`.  `Entry` sounds way too generic.